glass/src/lib/native/cpp/other/Log.cpp

```diff
@@ -65,6 +69,12 @@ void LogView::Display() {
     if (ImGui::Selectable("Clear")) {
       m_data->Clear();
     }
+    if (ImGui::Selectable("Copy to Clipboard")) {
+      const auto& buf = m_data->GetBuffer();
+      if (!buf.empty()) {
+        ImGui::SetClipboardText(buf.data());
```

Comment:
You need to null terminate the buffer.  If m_data was changed to be a `std::string` instead of a `vector<char>`, you could use `c_str()`.

Reply:
Alternatively you could have GetBuffer append a nul terminator before returning, or you could make a full copy of the buffer here.